metaflow/monitor.py

```diff
@@ -34,49 +36,59 @@ def terminate(self):
 
 
 class Monitor(NullMonitor):
-    def __init__(self, monitor_type, env, flow_name):
-        # type: (str) -> None
+    def __init__(self, monitor_type, env, **kwargs):
+        # type: (str, str, str) -> None
         self.sidecar_process = None
         self.monitor_type = monitor_type
-        self.env_info = env.get_environment_info()
-        self.env_info["flow_name"] = flow_name
+        self._context = {"env": env.get_environment_info()}
```

Comment:
Similar to the logger feedback - can the `AtlasMonitor` assume the responsibility for setting the right context from env and what not?

Reply:
yes, am fine moving both out. We had it this way already so my modus operandi was "don't remove stuff" but am happy to remove.